packages/cf-wallet.js/src/app-instance.ts

```diff
@@ -2,7 +2,7 @@ import {
   AppABIEncodings,
   AppInstanceInfo,
   AppInstanceJson,
-  CoinTransferInterpreterParams,
+  CoinTransferListOfListsInterpreterParams,
```

Comment:
Although this is accurate, it doesn't immediately convey why it's a list of lists.

Maybe something like `AssetBasedListOfCoinTransfersInterpreterParams` ?